tools/plugins/com.liferay.ide.layouttpl.core/src/com/liferay/ide/layouttpl/core/operation/NewLayoutTplDataModelProvider.java

```diff
@@ -62,6 +62,22 @@ private boolean checkDocrootFileExists( final IPath path )
         return false;
     }
 
+    private boolean checkoutDocrootFileNameCorrect( String filename,String type )
```

Comment:
Check the source formatting here, you can run the formatter on it to get it closer the right style.